tests/regression/test_star_pc.py

```diff
@@ -290,7 +290,64 @@ def test_vanka_equivalence(problem_type):
                        "mg_coarse_assembled_pc_factor_mat_solver_type": "mumps"}
 
     elif problem_type == "mixed":
-        return
+        base = UnitSquareMesh(5, 5, distribution_parameters=distribution_parameters, quadrilateral=True)
+        mh = MeshHierarchy(base, 1, distribution_parameters=distribution_parameters)
+        mesh = mh[-1]
+        V1 = VectorFunctionSpace(mesh, "CG", 2)
+        V2 = FunctionSpace(mesh, "CG", 1)
+        V = MixedFunctionSpace([V1, V2])
+
+        u = Function(V)
+        (z, p) = split(u)
+        (v, q) = split(TestFunction(V))
+
+        a = inner(grad(z), grad(v))*dx + inner(p, div(v))*dx + inner(q, div(z))*dx
```

Comment:
The two last terms would usually have a minus, but whatever (only changes sign of pressure)

Reply:
Fair enough.  Changed this while fixing the above